src/scrapers/mizrahi.ts

```diff
@@ -107,7 +108,7 @@ function convertTransactions(txns: ScrapedTransaction[]): Transaction[] {
       date: txnDate,
       processedDate: txnDate,
       originalAmount: row.MC02SchumEZ,
-      originalCurrency: SHEKEL_CURRENCY,
+      originalCurrency: row.kodMatbea ?? SHEKEL_CURRENCY,
```

Comment:
Hello, 
Few questions about this one.
1. Can you please provide an example of different values in `kodMatebea`?
2. it this currency of the original value or the charged one? meaning will you get something like 
```
{
MC02SchumEZ: 1,
kodMatbea: '$'
MC02SchumEZ: 3.1
}
```

Reply:
Hey,
the reason that this PR is in draft state is because I didn't finish doing the research,
thought I would get to it today but will probably look into it tomorrow.

Will keep you updated :)